README.rst

```diff
@@ -91,8 +91,6 @@ base operating system will allow (*allow* in this case means how many Virtual IP
 
 For more information please browse the project documentation:
 
-https://riptideio.github.io/pymodbus/
```

Comment:
I couldn't find the GitHub pages for the new name, so I just deleted this link

Reply:
yes this should be readthedocs, which is our official place for documentation.

hmmm, I might actually have missed that one.